src/storage.js

```diff
@@ -280,6 +195,32 @@ class AWSStorage {
     log.info(`Object uploaded to: ${this.bucket}/${key}`);
   }
 
+  /**
+   * Store an object contents, along with headers.
```

Comment:
The jsdoc for this method has been copied from `store(key, res)`, which surprisingly makes it only 50% wrong; it should probably say: `store some data along with optional metadata to S3` while the other method should say `store a *response* along with headers to S3`